pillar.example

```diff
@@ -1,17 +1,19 @@
 postgres:
   pg_hba.conf: salt://postgres/pg_hba.conf
-  commands:
-    initdb: service postgresql initdb
+  prepare_cluster:
+    user: root
+    command: service postgresql initdb
+    test: test -f /path/to/some/file
```

Comment:
The problem is, of course, that the path will be different on every OS (which is why it's overridden later).